### PR TITLE
fix: Make the workspace URL pretty

### PR DESCRIPTION
**Pull request by kylecarbs**
This adds the `@username/workspacename` format to the
workspace page!

The only oddity I've noticed is no underline for `Workspaces` anymore, but I'm not sure whether it should have been anyways!
image

### CODE REVIEW

**Review by greyscaled — APPROVED** [review]

**Comment by presleyp** [comment]
I think it's important for the user to have a cue about which page they're on (that underline we used to have). Here's the docs on it https://v5.reactrouter.com/web/api/NavLink, you probably have to update `NavBarView.tsx`

**Comment by presleyp** [comment]
@kylecarbs but yeah I see your point, maybe it shouldn't say you're on the workspaces page when you're on the workspace page. Not sure what's typical there!

**Comment by kylecarbs** [comment]
Going to merge for now. @f0ssel and I chatted a while back and he seemed to be good with this route, but please comment if I'm mistaken!

File: site/webpack.dev.ts [hunk 1]
```diff
@@ -63,7 +63,7 @@ const config: Configuration = {
     port: process.env.PORT || 8080,
     proxy: {
       "/api": {
-        target: "http://localhost:3000",
+        target: "https://dev.coder.com",
```
Comment (AbhineetJain) [inline site/webpack.dev.ts:66]: @kylecarbs Is this an expected change? Broke my local development dummy login creds.

**Comment by AbhineetJain** [comment]
I also found that redirection after creating a new workspace via the UI is broken. I am assuming we missed adding that in this PR.

File: site/webpack.dev.ts (hunk 1 above)
Comment (kylecarbs) [reply site/webpack.dev.ts:66]: I'm going to make this a flag so this never gets in again!

**Comment by kylecarbs** [comment]
Ahh yes. I'll fix that in my other PR for schedules, feel free to review that!

**Pull request merged**